addons/calendar/static/tests/activity_tests.js

```diff
@@ -35,3 +35,42 @@ QUnit.test("activity click on Reschedule", async () => {
     await click(".btn", { text: "Reschedule" });
     await contains(".o_calendar_view");
 });
+
+QUnit.test("activity with event click on cancel", async (assert) => {
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({});
```

Comment:
Empty objects when creating records is something we'd like to avoid to stay as close as possible to the real flow. In real life, a partner would not be created with empty data. Giving him a name at least would be better.